<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>accountDialog</class>
 <widget class="QMainWindow" name="accountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arkadaş Listesi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QListWidget" name="listWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>279</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHesap">
    <property name="title">
     <string>Hesap</string>
    </property>
    <addaction name="actionAdd_Friend"/>
   </widget>
   <addaction name="menuHesap"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_Friend">
   <property name="text">
    <string>Arkadaş Ekle</string>
   </property>
  </action>
  <action name="action_k">
   <property name="text">
    <string>Çıkış</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
